src/duplo_resource/s3.py

```diff
@@ -0,0 +1,73 @@
+from duplocloud.client import DuploClient
+from duplocloud.resource import DuploTenantResourceV2
+from duplocloud.errors import DuploError
+from duplocloud.commander import Command, Resource
+import duplocloud.args as args
+
+
+@Resource("s3")
+class DuploS3(DuploTenantResourceV2):
```

Comment:
This is a v3 api not v2
Hence the endpoint: `v3/subscriptions`
Extend this class instead `DuploTenantResourceV3`